### PR TITLE
ECER-1077: Application Summary tasks for 400 hour work experience

### PR DESCRIPTION
**Pull request by qsl-deepak**
## Title
ECER-1077: Application Summary tasks for 400 hour work experience

## Description

- Added/adjusted application Summary tasks for 400 hour work experience
- Added/adjusted manage 400 hour work experience references

## Related Jira Issue(s)

- ECER-1077
- ECER-2352

## Checklist
- [x] I have tested these changes locally.
- [x] Changes to backend endpoints are covered by passing integration tests.
- [ ] I have added or updated the necessary documentation.

## Screenshots (if applicable)

localhost_5121_manage-application_896958d8-9ab4-439a-9ed7-6e832c31b413_ (3)

localhost_5121_manage-application_896958d8-9ab4-439a-9ed7-6e832c31b413_ (1)

localhost_5121_manage-application_896958d8-9ab4-439a-9ed7-6e832c31b413_ (2)

### CODE REVIEW

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ManageWorkExperienceReferenceList.vue [hunk 1]
```diff
@@ -96,10 +114,16 @@ export default defineComponent({
     const route = useRoute();
 
     const applicationStatus = (await getApplicationStatus(route.params.applicationId.toString()))?.data;
+    const certifications = (await getCertifications()).data!;
```
Comment (pcatkins) [inline src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ManageWorkExperienceReferenceList.vue:117]: can we use the `latestCertification `from `store/certification.ts`? Or do you need refresh here?

Comment (pcatkins) [reply src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ManageWorkExperienceReferenceList.vue:117]: Same question for `applicationStatus`.

**Review by pcatkins — CHANGES_REQUESTED** [review]
See comments.

**Comment by farzadnadiri** [comment]
backend looks good

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ManageWorkExperienceReferenceList.vue (hunk 1 above)
Comment (qsl-deepak) [reply src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ManageWorkExperienceReferenceList.vue]: took latestCertification from store/certification. keping applicationStatus as an API call for now.

**Review by pcatkins — APPROVED** [review]

**Pull request merged**